docs/product-installation-guide/src/main/asciidoc/clustering-dm-install-proc.adoc

```diff
@@ -8,10 +8,10 @@ A {EAP} 7.1 cluster is configured and Elasticsearch and AMQ Broker 7.1 are insta
 
 .Procedure
 
-. Install {CENTRAL} on each node of the cluster. See _INSTALLING AND CONFIGURING RED HAT PROCESS AUTOMATION MANAGER ON EAP 7.1_ for information about installing {CENTRAL}.
-. On each node of the cluster, edit the following properties in the `<EAP_HOME>/standalone/configuration/standalone.xml` file, where:
-* `<NIOGIT_DIR>` is the location git repositories stored under ``.niogit`, on an NFS mounted partition.
-* `<MAIN_NODE_IP>` is the IP address of the node where Elasticseach and AMQ Broker are installed.
+. Install {CENTRAL} on each node of the cluster. See _Installing and configuring {PRODUCT} on {EAP} 7.1_ for information about installing {CENTRAL}.
+. On each node of the cluster, edit the following properties in the `<EAP_HOME>/domain/configuration/domain.xml` file, where:
```

Comment:
Comes up multiple times. A case where I personally am great with this and see this as pretty standard and clear. But I think IBM is against this saying "where:" with a list of variables, with the same logic/rule of not ending incomplete sentences with a colon. I think they would probably just end with ".. file:" with the list of things to edit. In this case the "where" is possibly just unnecessary. In other cases, I think IBM would say "Replace the following values:" or the like. Again, using "where:" followed by replacable variables is clear and standard in the industry, and is even recommended in other tech style manuals (Sun Microsystems and Microsoft for instance both do). I'm just making the IBM plug, much to your and my dismay :)